src/picosvg/svg_pathops.py

```diff
@@ -111,6 +120,12 @@ def intersection(*svg_cmd_seqs) -> SVGCommandGen:
     return _do_pathop(pathops.PathOp.INTERSECTION, svg_cmd_seqs)
 
 
+def remove_overlaps(svg_cmds: SVGCommandSeq, fill_rule: str) -> SVGCommandGen:
```

Comment:
Somewhere, perhaps here, we should elaborate on what we're doing and why. Maybe cite the article you had about winding, etc. Assume a solid SWE who doesn't know about winding is reading basically.